### PR TITLE
:sparkles: hasKey selector

### LINKED ISSUE #83: new selector: hasKey
@bpetetot

### PR DESCRIPTION
**Pull request by fabienjuif**
fixes #83

### CODE REVIEW

**Comment by fabienjuif** [comment]
I will do this on publishing

File: src/selectors.js [hunk 1]
```diff
@@ -31,3 +31,9 @@ export const getBy = options => (propertyPath, values) => (state) => {
   if (Array.isArray(values)) return data.filter(d => values.includes(at(d, propertyPath)[0]))
   return data.filter(d => values === at(d, propertyPath)[0])
 }
+
+export const hasKey = options => key => (state) => {
+  const keys = getKeys(options)(state)
+
+  return !!keys.includes(key)
```
Comment (bpetetot) [inline src/selectors.js]: Is there a reason to use `!!`, I believed `includes` already returns `true` or `false`
I think you can do:
```es6
return keys.includes(key)
```

Comment (fabienjuif) [reply src/selectors.js]: There is no reason this is something I do everytime I need a boolean now.
But you are right in this case. I'll fix that

**Comment by fabienjuif** [comment]
@bpetetot ready for review please

**Pull request merged**